src/Generators/ModelGenerator.php

```diff
@@ -2,6 +2,7 @@
 
```

Comment:
Thought about creating a stand alone generateEloquent class. Perhaps in next rev.